### DISCUSSION TITLE
MajorGridlineThickness doesn't work with floats?

### DISCUSSION THREAD

**Discussion by bebenins** [oxyplot/oxyplot#1897] [Q&A]
Hello, I'm trying to set stroke thickness of `MajorGridlineThickness` to 0.2 and when I do, it's simply not even there.  Is is possible to make MajorGridline stand out a little less?
Thanks!

**Comment by VisualMelon** [comment]
The behavour of stroke thickness will depend on the `EdgeRenderingMode` set on the `Axis`. Some will disable anti-aliasing.

One alternative is to make the grid line color translucent or otherwise closer to the background.
